Changelog entry for v0.3.0 (#887)
GenericMappingTools/pygmt: doc/changes.rst

* Add slash command '/test-gmt-dev' to test GMT dev version (#831)
* Add tests for grdfilter (#809)
* Improve the error message when PyGMT fails to load the GMT library (#814)
* Add tests for GMTInvalidInput for figure.py (#810)
* Split meca into a standalone module from base_plotting.py (#686)
* Update CONTRIBUTING.md with formatting conventions (#775)
* Use args_in_kwargs for basemap error raising (#797)
* Add docformatter to format plain text in docstrings (#642)
* Add GMTInvalidInput Error for Figure.coast (#787)
* Add args_in_kwargs function (#791)
* Add a CI job testing GMT master branch on Windows (#756)
* Add isort to sort imports alphabetically (#745)
* Convert relative imports to absolute imports (#754)
* Add a Makefile target 'distclean' for deleting project metadata files (#744)
* Add test test_basemap_map_scale (#739)
* Check files for UNIX-style line breaks and 644 permission (#736)
* Rename vercel configuration file from now.json to vercel.json (#738)
* Migrate pytest configurations to pyproject.toml (#725)
* Migrate documentation deployment from Travis CI to GitHub Actions (#713)
* Switch from versioneer to setuptools-scm (#695)

> * Migrate coverage configurations to pyproject.toml (#667)

Reviewer: Roughly group the changes into categories (CI, source codes, documentation, tests, and misc). The changes in the 'misc' category may be moved into "Enhancements" instead.

**NOTE**: Need to remove the blank lines before committing the changes.

```suggestion
* Add dependabot to keep GitHub Actions up to date (#861)
* Skip workflows in PRs if only non-source-code files are changed (#839)
* Add slash command '/test-gmt-dev' to test GMT dev version (#831)
* Check files for UNIX-style line breaks and 644 permission (#736)
* Rename vercel configuration file from now.json to vercel.json (#738)
* Add a CI job testing GMT master branch on Windows (#756)
* Migrate documentation deployment from Travis CI to GitHub Actions (#713)

* Move Figure.meca() into a standalone module (#686)
* Move plotting functions to separate modules (#808)
* Move non-plotting modules to separate modules (#832)
* Add isort to sort imports alphabetically (#745)
* Convert relative imports to absolute imports (#754)
* Switch from versioneer to setuptools-scm (#695)

* Add authorship policy (#726)
* Add docformatter to format plain text in docstrings (#642)
* Update CONTRIBUTING.md with formatting conventions (#775)

* Migrate pytest configurations to pyproject.toml (#725)
* Migrate coverage configurations to pyproject.toml (#667)
* Show test execution times in pytest (#835)
* Add tests for grdfilter (#809)
* Add tests for GMTInvalidInput of Figure.savefig() and Figure.show() (#810)
* Add args_in_kwargs function (#791)
* Add a Makefile target 'distclean' for deleting project metadata files (#744)
* Add a test Figure.basemap() map_scale (#739)

* Improve the error message when PyGMT fails to load the GMT library (#814)
* Add GMTInvalidInput Error for Figure.coast (#787)
* Use args_in_kwargs for basemap error raising (#797)
```

Reviewer: See the new comment https://github.com/GenericMappingTools/pygmt/pull/887#discussion_r575930143.

Changelog
=========

Release v0.3.0 (2021/02/15)
---------------------------

.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.4522136.svg
   :alt: Digital Object Identifier for the Zenodo archive
   :target: https://doi.org/10.5281/zenodo.4522136

Highlights

* 🎉 **Third minor release of PyGMT** 🎉
* Wrap inset (#788) for making overview maps and subplot (#822) for multi-panel figures
* Apply standardized formatting conventions (#775) across most documentation pages
* Drop Python 3.6 support (#699) so PyGMT now requires Python 3.7 or newer

New Features

* Wrap grd2cpt (#803)
* Let Figure.text() support record-by-record transparency (#716)
* Provide basic support for FreeBSD (#700, #878)

Enhancements

* Let load_earth_relief() support the 'region' parameter for all resolutions (#873)
* Improve how PyGMT finds the GMT library (#702)
* Add dcw (#765) and lakes (#781) aliases to Figure.coast

> * Add the shading parameter to Figure.colorbar (#752)

Reviewer: ```suggestion
* Add the shading parameter to Figure.colorbar (#752)
* Improve the error message when PyGMT fails to load the GMT library (#814)
* Add GMTInvalidInput Error for Figure.coast (#787)
```

Documentation

> 

Reviewer: ```suggestion
* Add authorship policy (#726)
```

* Create gallery example for datetime inputs (#779)
* Add an example for plotting multi-parameter symbols (#772)
* Add a gallery example for the fig.image() method (#777)
* Add a gallery example of setting line colors with a custom CPT (#774)
* Add a table for compatibility of PyGMT with Python and GMT (#763)
* Add description for the "columns" arguments (#766)
* Add table for projections (#753)
* Add more gallery examples for projections (#761, #721, #757, #723, #762, #742)
* Add projection description for Lambert Azimuthal Equal-Area (#760)
* Change text when GMTInvalidInput error is raised for basemap (#729)
* Add non-geographic plots to projection gallery (#728)
* Adding cylindrical projections to gallery (#727)
* Add tutorials, for adding a map title (#720), plotting Earth relief (#712), 3D perspective image (#743), contour maps (#705), plotting lines tutorial (#741), region arguments (#800)
* Add frame and cmap to the colorbar parameters list (#709)
* Add frame as a parameter on the grdview documentation (#707)

Bug Fixes

* Fix a bug of Figure.text() when "text" is a non-string array (#724)
* Fix the error message when IPython is not available (#701)

> Maintenance
> 
> * Add dependabot to keep GitHub Actions up to date (#861)
> * Add authorship policy (#726)
> * Move plotting functions to separate modules (#808)
> * Skip workflows in PRs if only non-source-code files are changed (#839)
> * Move non-plotting modules to separate files in src directory (#832)
> * Show test execution times in pytest (#835)
> * Add slash command '/test-gmt-dev' to test GMT dev version (#831)
> * Add tests for grdfilter (#809)
> * Improve the error message when PyGMT fails to load the GMT library (#814)
> * Add tests for GMTInvalidInput for figure.py (#810)
> * Split meca into a standalone module from base_plotting.py (#686)
> * Use args_in_kwargs for basemap error raising (#797)
> * Add docformatter to format plain text in docstrings (#642)
> * Add GMTInvalidInput Error for Figure.coast (#787)
> * Add args_in_kwargs function (#791)
> * Add a CI job testing GMT master branch on Windows (#756)
> * Add isort to sort imports alphabetically (#745)
> * Convert relative imports to absolute imports (#754)
> * Add a Makefile target 'distclean' for deleting project metadata files (#744)
> * Add test test_basemap_map_scale (#739)
> * Check files for UNIX-style line breaks and 644 permission (#736)
> * Rename vercel configuration file from now.json to vercel.json (#738)
> * Migrate pytest configurations to pyproject.toml (#725)
> * Migrate documentation deployment from Travis CI to GitHub Actions (#713)
> * Switch from versioneer to setuptools-scm (#695)
> * Migrate coverage configurations to pyproject.toml (#667)

Reviewer: Roughly group the changes into categories (CI, source codes, documentation, tests, and misc), and update description of a few changes. The changes in the 'misc' category (the last three entries) may be moved into "Enhancements" instead.

**NOTE**: Need to remove the blank lines before committing the changes.

```suggestion
* Add dependabot to keep GitHub Actions up to date (#861)
* Skip workflows in PRs if only non-source-code files are changed (#839)
* Add slash command '/test-gmt-dev' to test GMT dev version (#831)
* Check files for UNIX-style line breaks and 644 permission (#736)
* Rename vercel configuration file from now.json to vercel.json (#738)
* Add a CI job testing GMT master branch on Windows (#756)
* Migrate documentation deployment from Travis CI to GitHub Actions (#713)

* Move Figure.meca() into a standalone module (#686)
* Move plotting functions to separate modules (#808)
* Move non-plotting modules to separate modules (#832)
* Add isort to sort imports alphabetically (#745)
* Convert relative imports to absolute imports (#754)
* Switch from versioneer to setuptools-scm (#695)

* Add authorship policy (#726)
* Add docformatter to format plain text in docstrings (#642)

* Migrate pytest configurations to pyproject.toml (#725)
* Migrate coverage configurations to pyproject.toml (#667)
* Show test execution times in pytest (#835)
* Add tests for grdfilter (#809)
* Add tests for GMTInvalidInput of Figure.savefig() and Figure.show() (#810)
* Add args_in_kwargs function (#791)
* Add a Makefile target 'distclean' for deleting project metadata files (#744)
* Add a test Figure.basemap() map_scale (#739)

* Improve the error message when PyGMT fails to load the GMT library (#814)
* Add GMTInvalidInput Error for Figure.coast (#787)
* Use args_in_kwargs for basemap error raising (#797)
```

Reviewer 2: I think the maintenance/documentation PR 726 could probably fit under the actual documentation section, but keep 642 in maintenance. Agree that 814 and 787 could be under enhancements, but maybe not 787 (keep it under maintenance) since that is non-user facing.

Feel free to make the change, I'm gonna have dinner first.

Changelog
=========

Release v0.3.0 (2021/02/15)
---------------------------

.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.4522136.svg
   :alt: Digital Object Identifier for the Zenodo archive
   :target: https://doi.org/10.5281/zenodo.4522136

Highlights

> *

Reviewer 2: We'll need to add 3 to 4 lines for the highlights section here. How about:

```suggestion
* 🎉 **Third minor release of PyGMT** 🎉
* Wrap inset (#788) for making overview maps and subplot (#822) for multi-panel figures
* Apply standardized formatting conventions (#775) across most documentation pages
* Drop Python 3.6 support (#699) so PyGMT now requires Python 3.7 or newer
```
Feel free to suggest edits in the wording. I'm also thinking whether to mention the upcoming drop in support for GMT 6.1.1 (once GMT 6.2.0 is out, something for PyGMT v0.4.0) and/or NEP29 adoption (#690). Might depend on whether we want a v0.3.1 release?

Reviewer: > I'm also thinking whether to mention the upcoming drop in support for GMT 6.1.1 (once GMT 6.2.0 is out, something for PyGMT v0.4.0) and/or NEP29 adoption

Don't try to predict the future.

Reviewer 2: Ok, will surprise our users then :smile_cat:

New Features

* Wrap grd2cpt (#803), inset (#788), and subplot (#822)
* Let Figure.text() support record-by-record transparency (#716)

> * Provide basic support for FreeBSD (#700)

Reviewer: ```suggestion
* Provide basic support for FreeBSD (#700, #878)
```

Enhancements

> * Let load_earth_relief() support 'region' argument for all resolutions (#873)

Reviewer: ```suggestion
* Let load_earth_relief() support the 'region' parameter for all resolutions (#873)
```

> * Detect ghostscript and default image viewer in FreeBSD (#878)

Reviewer: ```suggestion
```

* Improve how PyGMT finds the GMT library (#702)
* Add dcw (#765) and lakes (#781) aliases to Figure.coast

> * Add the shading parameter to colorbar() in base_plotting.py (#752)

Reviewer: ```suggestion
* Add the shading parameter to Figure.colorbar (#752)
```

Documentation

* Update the docstrings in the plotting modules (#881)
* Update the doc strings in the non-plotting modules (#882)
* Add alias annotation (A) to Figure.contour() (#883)
* Revise pygmt development installation instructions (#865)
* Update the docstrings of common aliases (#862)
* Add common alias panel (-c) to all plotting functions (#853)

> * Update forum link to pygmt specific category (#858)

Reviewer: ```suggestion
```

Reviewer 2: I'll add a skip-changelog label to this PR.

* Wrap grdinfo aliases (#799)
* Add sphinx-copybutton extension to easily copy codes (#838)

> * Add additional video link in text for clarity (#836)

Reviewer: ```suggestion
```

Reviewer 2: Same, I'll add a skip-changelog label to this PR (nothing against Liam).

* Choosing the thumbnail figures in tutorials (#826)
* Add a gallery example for logo (#823)
* Update axis label explanation in frames tutorial (#820)
* Update GMT arguments in coast doc strings (#798)
* Add guidelines for types of tests to write (#796)
* Recommend using SI units in documentation (#795)
* Create gallery example for datetime inputs (#779)

> * Update GMT argument in the azimuthal (#776), conic (#780), cylindric (#789), and miscellaneous (#790) projections gallery to doc standards

Reviewer: ```suggestion
```

Reviewer 2: Assuming that we just use the line at https://github.com/GenericMappingTools/pygmt/pull/887/files#r575914421? If so, then we should add skip-changelog tags to all of these.

Reviewer: Do we really need to add "skip-changelog" label to this PRs? The "skip-changelog" label won't have any effects for future release notes, right?

Reviewer 2: Mm yes. It's more just in case we get another PR merged in before this changelog PR is finalized and merged.

Reviewer: Even though more PRs are merged before we finalize this changelog, we won't go to the "draft release notes" and start everything again. So labeling "skip-changelog" now seems useless to me.

* Add an example for plotting multi-parameter symbols (#772)
* Add a gallery example for the fig.image() method (#777)
* Add a gallery example of setting line colors with a custom CPT (#774)
* Add a table for compatibility of PyGMT with Python and GMT (#763)
* Add description for the "columns" arguments (#766)
* Add table for projections (#753)

> * Add cylindric (#761), conic projection (#721, #757), miscellaneous projections descriptions (#723, #762), logarithmic and power projections (#742) to gallery

Reviewer: ```suggestion
* Add more gallery examples for projections (#761, #721, #757, #723, #762, #742)
```

* Improve the error message when PyGMT fails to load the GMT library (#814)
* Add tests for GMTInvalidInput for figure.py (#810)
* Split meca into a standalone module from base_plotting.py (#686)
* Update CONTRIBUTING.md with formatting conventions (#775)
* Use args_in_kwargs for basemap error raising (#797)
* Add docformatter to format plain text in docstrings (#642)
* Add GMTInvalidInput Error for Figure.coast (#787)
* Add args_in_kwargs function (#791)
* Add a CI job testing GMT master branch on Windows (#756)
* Add isort to sort imports alphabetically (#745)
* Convert relative imports to absolute imports (#754)
* Add a Makefile target 'distclean' for deleting project metadata files (#744)
* Add test test_basemap_map_scale (#739)
* Check files for UNIX-style line breaks and 644 permission (#736)
* Rename vercel configuration file from now.json to vercel.json (#738)
* Migrate pytest configurations to pyproject.toml (#725)
* Migrate documentation deployment from Travis CI to GitHub Actions (#713)
* Switch from versioneer to setuptools-scm (#695)
* Migrate coverage configurations to pyproject.toml (#667)

Deprecations

* Drop Python 3.6 support (#699)

Contributors

> @liamtoney, @meghanrjones, @michaelgrund, @seisman, @weiji14 and @willschlitzer

Reviewer 2: ```suggestion
People with a "+" by their name contributed a patch for the first time.

* `Dongdong Tian <https://github.com/seisman>`__
* `Liam Toney <https://github.com/liamtoney>`__
* `Meghan Jones <https://github.com/meghanrjones>`__ +
* `Michael Grund <https://github.com/michaelgrund>`__ +
* `Wei Ji Leong <https://github.com/weiji14>`__
* `Will Schlitzer <https://github.com/willschlitzer>`__ +
```

Two thoughts here:
1. Should we sort alphebetically by name (as above), or by number of commits (old style before).
2. Some projects (e.g. [geopandas](https://github.com/geopandas/geopandas/releases/tag/v0.8.0)) add a little note with a plus (+) to indicate people's first contribution, should we add it here?

Reviewer: > Two thoughts here:
> 
> 1. Should we sort alphebetically by name (as above), or by number of commits (old style before).

I'm fine with both, but prefer "number of commits".

> 2. Some projects (e.g. [geopandas](https://github.com/geopandas/geopandas/releases/tag/v0.8.0)) add a little note with a plus (+) to indicate people's first contribution, should we add it here?

I saw it in many projects, but is there an easy way to know if someone makes his/her first contribution to this release?

Reviewer 2: > > 1. Some projects (e.g. [geopandas](https://github.com/geopandas/geopandas/releases/tag/v0.8.0)) add a little note with a plus (+) to indicate people's first contribution, should we add it here?
> 
> I saw it in many projects, but is there an easy way to know if someone makes his/her first contribution to this release?

Other than remembering people, there's probably a complicated git command that could do it. Happy to leave it without the plus.

> I'm fine with both, but prefer "number of commits".

I thought your name would be on top either way, but `git shortlog HEAD...v0.2.1 -sne` gives:

```
    58  Will Schlitzer <schlitzer90@gmail.com>
    54  Dongdong Tian <seisman.info@gmail.com>
     9  Wei Ji <23487320+weiji14@users.noreply.github.com>
     7  Michael Grund <michael_grund@gmx.de>
     6  dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
     3  Liam Toney <38269494+liamtoney@users.noreply.github.com>
     3  Meghan Jones <meghanj@hawaii.edu>
     3  Wei Ji <weiji.leong@vuw.ac.nz>
```
and so the order would be:
```suggestion
* `Will Schlitzer <https://github.com/willschlitzer>`__
* `Dongdong Tian <https://github.com/seisman>`__
* `Wei Ji Leong <https://github.com/weiji14>`__
* `Michael Grund <https://github.com/michaelgrund>`__
* `Liam Toney <https://github.com/liamtoney>`__
* `Meghan Jones <https://github.com/meghanrjones>`__
```

Reviewer: Looks better to me.

Reviewer 2: Alright, I'll commit the change, but we should document this (order by number of commits) for when we deal with #885.

Reviewer: Running `git shortlog HEAD...v0.2.1 -sne` on the master branch gives me:
```
    54  Dongdong Tian <seisman.info@gmail.com>
    54  Will Schlitzer <schlitzer90@gmail.com>
     9  Wei Ji <23487320+weiji14@users.noreply.github.com>
     7  Michael Grund <michael_grund@gmx.de>
     6  dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
     3  Liam Toney <38269494+liamtoney@users.noreply.github.com>
     3  Meghan Jones <meghanj@hawaii.edu>
```

Not trying to be listed at the top, just curious why our results are different 😂

Reviewer 2: Hmm, I was probably on a different branch (which explains why it had my unversity vuw.ac.nz email). I think you'll get bumped down after @willschlitzer merges this PR though :rofl:

Author: Dare I suggest that as a GMT developer and long-time PyGMT maintainer, I think @seisman has definitely earned the first spot!

Changelog
=========

Release v0.3.0 (2021/02/15)
---------------------------

.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.4522136.svg
   :alt: Digital Object Identifier for the Zenodo archive
   :target: https://doi.org/10.5281/zenodo.4522136

Highlights

* 🎉 **Third minor release of PyGMT** 🎉
* Wrap inset (`#788 <https://github.com/GenericMappingTools/pygmt/pull/788>`__) for making overview maps and subplot (`#822 <https://github.com/GenericMappingTools/pygmt/pull/822>`__) for multi-panel figures
* Apply standardized formatting conventions (`#775 <https://github.com/GenericMappingTools/pygmt/pull/775>`__) across most documentation pages
* Drop Python 3.6 support (`#699 <https://github.com/GenericMappingTools/pygmt/pull/699>`__) so PyGMT now requires Python 3.7 or newer

New Features

* Wrap grd2cpt (`#803 <https://github.com/GenericMappingTools/pygmt/pull/803>`__)
* Let Figure.text() support record-by-record transparency (`#716 <https://github.com/GenericMappingTools/pygmt/pull/716>`__)

> * Provide basic support for FreeBSD (#700, #878)

Reviewer 2: Seems like the `sed` command missed a few, looks like we forgot to update it after https://github.com/GenericMappingTools/pygmt/pull/683#discussion_r523388058. Try
```
sed -i.bak -E 's$\(#([0-9]*)\)$(`#\1 <https://github.com/GenericMappingTools/pygmt/pull/\1>`__)$g' changes.rst
```

Author: Manually switched the formatting for some of the PR refs; the problem was when they were both in a set of parentheses.

> * Provide basic support for FreeBSD (`#700 <https://github.com/GenericMappingTools/pygmt/pull/700>`__) (`#878 <https://github.com/GenericMappingTools/pygmt/pull/878>`__)

Reviewer 2: ```suggestion
* Provide basic support for FreeBSD (`#700 <https://github.com/GenericMappingTools/pygmt/pull/700>`__, `#878 <https://github.com/GenericMappingTools/pygmt/pull/878>`__)
```

Enhancements

* Let load_earth_relief() support the 'region' parameter for all resolutions (#873)
* Improve how PyGMT finds the GMT library (#702)
* Add common alias panel (-c) to all plotting functions (#853)

> * Add aliases dcw (#765) and lakes (#781) to Figure.coast
> * Add alias shading to Figure.colorbar (#752)
> * Add alias annotation (A) to Figure.contour() (#883)
> * Wrap Figure.grdinfo() aliases (#799)
> * Add aliases frame and cmap to Figure.colorbar() (#709)
> * Add alias frame to Figure.grdview() (#707)
> * Improve the error message when PyGMT fails to load the GMT library (#814)
> * Add GMTInvalidInput Error for Figure.coast (#787)

Reviewer: Consistently use `Figure.function()` (with parentheses） or `Figure.function` (without parentheses). 
```suggestion
* Add aliases dcw (#765) and lakes (#781) to Figure.coast()
* Add alias shading to Figure.colorbar() (#752)
* Add alias annotation (A) to Figure.contour() (#883)
* Wrap Figure.grdinfo() aliases (#799)
* Add aliases frame and cmap to Figure.colorbar() (#709)
* Add alias frame to Figure.grdview() (#707)
* Improve the error message when PyGMT fails to load the GMT library (#814)
* Add GMTInvalidInput error to Figure.coast() (#787)
```

Reviewer 2: I think without parentheses is better.

Reviewer: I'm OK with both. Feel free to make any changes.

Reviewer 2: There's still a lot of parentheses in the changelog @willschlitzer, could you remove all of them (just the v0.3.0 ones) to be consistent?

Documentation

* Add authorship policy (#726)
* Update PyGMT development installation instructions (#865)
* Add a tutorial for adding a map title (#720)
* Add a tutorial for plotting Earth relief (#712)
* Add a tutorial for 3D perspective image (#743)
* Add a tutorial for contour maps (#705)
* Add a tutorial for plotting lines
* Add a turorial for the region argument (#800)
* Add a gallery example for datetime inputs (#779)
* Add a gallery example for Figure.logo() (#823)
* Add a gallery example for plotting multi-parameter symbols (#772)
* Add a gallery example for Figure.image() (#777)
* Add a gallery example for setting line colors with a custom CPT (#774)
* Add more gallery examples for projections (#761, #721, #757, #723, #762, #742, #728, #727)
* Update the docstrings in the plotting modules (#881)
* Update the docstrings in the non-plotting modules (#882)
* Update Figure.coast() docstrings (#798)
* Update the docstrings of common aliases (#862)
* Add sphinx-copybutton extension to easily copy codes (#838)

> * Choosing the thumbnail figures in tutorials (#826)

Reviewer: ```suggestion
* Choose the best figures in tutorials for thumbnails (#826)
```

> * Add a turorial for the region argument (`#800 <https://github.com/GenericMappingTools/pygmt/pull/800>`__)

Reviewer 2: ```suggestion
* Add a tutorial for the region argument (`#800 <https://github.com/GenericMappingTools/pygmt/pull/800>`__)
```
Apologies, spoke too soon. I'll commit this and spare you the torture.

> * Add a gallery example for Figure.logo() (`#823 <https://github.com/GenericMappingTools/pygmt/pull/823>`__)

Reviewer 2: ```suggestion
* Add a gallery example for Figure.logo (`#823 <https://github.com/GenericMappingTools/pygmt/pull/823>`__)
```

Can you remove all the parentheses as mentioned in https://github.com/GenericMappingTools/pygmt/pull/887#discussion_r576008258?

Author: Fixed it in 98097bc

> * Add more gallery examples for projections (`#761 <https://github.com/GenericMappingTools/pygmt/pull/761>`__) (`#721 <https://github.com/GenericMappingTools/pygmt/pull/721>`__) (`#757 <https://github.com/GenericMappingTools/pygmt/pull/757>`__) (`#723 <https://github.com/GenericMappingTools/pygmt/pull/723>`__) (`#762 <https://github.com/GenericMappingTools/pygmt/pull/762>`__) (`#742 <https://github.com/GenericMappingTools/pygmt/pull/742>`__) (`#728 <https://github.com/GenericMappingTools/pygmt/pull/728>`__) (`#727 <https://github.com/GenericMappingTools/pygmt/pull/727>`__)

Reviewer 2: ```suggestion
* Add more gallery examples for projections (`#761 <https://github.com/GenericMappingTools/pygmt/pull/761>`__, `#721 <https://github.com/GenericMappingTools/pygmt/pull/721>`__, `#757 <https://github.com/GenericMappingTools/pygmt/pull/757>`__, `#723 <https://github.com/GenericMappingTools/pygmt/pull/723>`__, `#762 <https://github.com/GenericMappingTools/pygmt/pull/762>`__, `#742 <https://github.com/GenericMappingTools/pygmt/pull/742>`__, `#728 <https://github.com/GenericMappingTools/pygmt/pull/728>`__, `#727 <https://github.com/GenericMappingTools/pygmt/pull/727>`__)
```
